tests/sample/data_usage/associatetoflow.c

```diff
@@ -0,0 +1,27 @@
+#include <stdbool.h>
+#include <stdint.h>
+#include "ebpf_helpers.h"
+#include "ebpf_nethooks.h"
```

Comment:
I believe you need to combine two ebpf programs together so that they can share the maps.

Reply:
We only needed to combine the programs together so that the second eBPF program can access the flowId from the first eBPF program. Since we are doing a 2-map approach, combining is not necessary anymore.